docs/standard/native-interop/index.md

```diff
@@ -1,27 +1,26 @@
+The following articles dive a little bit deeper into all three ways of doing "native interoperability" that are available using .NET.
 
-There are a few of reasons why you would want to call into native code:
+There are a few of reasons why you'd want to call into native code:
 
-*   Operating Systems come with a large volume of APIs that are not present in the managed class libraries. A prime example for this would be access to hardware or operating system management functions.
+*   Operating Systems come with a large volume of APIs that aren't present in the managed class libraries. A prime example for this would be access to hardware or operating system management functions.
 *   Communicating with other components that have or can produce C-style ABIs (native ABIs). This covers, for example, Java code that is exposed via [Java Native Interface (JNI)](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/) or any other managed language that could produce a native component.
 *   On Windows, most of the software that gets installed, such as Microsoft Office suite, registers COM components that represent their programs and allow developers to automate them or use them. This also requires native interoperability.
 
-Of course, the list above does not cover all of the potential situations and scenarios in which the developer would want/like/need to interface with native components. .NET class library, for instance, uses the native interoperability support to implement a fair number of its APIs, like console support and manipulation, file system access and others. However, it is important to note that there is an option, should one need it.
+Of course, the list above doesn't cover all of the potential situations and scenarios in which the developer would want/like/need to interface with native components. .NET class library, for instance, uses the native interoperability support to implement a fair number of its APIs, like console support and manipulation, file system access and others. However, it is important to note that there is an option, should one need it.
 
 > [!NOTE]
-> Most of the examples in this section will be presented for all three supported platforms for .NET Core (Windows, Linux and macOS). However, for some short and illustrative examples, just one sample is shown that uses Windows filenames and extensions (that is, "dll" for libraries). This does not mean that those features are not available on Linux or macOS, it was done merely for convenience sake.
+> Most of the examples in this section will be presented for all three supported platforms for .NET Core (Windows, Linux and macOS). However, for some short and illustrative examples, just one sample is shown that uses Windows filenames and extensions (that is, "dll" for libraries). This doesn't mean that those features aren't available on Linux or macOS, it was done merely for convenience sake.
```

Comment:
If I recall, the proper casing should be "MacOS"

Reply:
@AaronRobinsonMSFT @jkoritzinsky 
The Apple website uses "macOS":
https://www.apple.com/macos/mojave/

The Microsoft website also use "macOS" spelling:
https://dotnet.microsoft.com/download/dotnet-core/2.1

Reply:
I'll switch it back to lowercase then.

Reply:
@pkulikov The official Mac site makes sense to me. The fact that a Microsoft managed site does it is not a good justification, especially when it is a dotnet one and so much of our doc isn't properly stylized :-/ Thanks for the Mac link!